Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -960,6 +960,16 @@ namespace AzToolsFramework
         return m_domPath;
     }
 
+    bool DPERowWidget::HasChildRows() const
+    {
+        return !m_domOrderedChildren.empty();
+    }
+
+    int DPERowWidget::GetLevel() const
+    {
+        return aznumeric_cast<int>(m_domPath.size()) - 1;
```

Comment:
I think m_depth has this same info. Either use that or remove it and use GetLevel everywhere please :)

Reply:
Done. Not sure what happened there! I completely missed DPERowWidget::m_depth when I was looking for it. Might have found the layout m_depth and given up.